test/integration/integration_helpers.go

```diff
@@ -80,83 +80,121 @@ spec:
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: {{ .OperatorName }}-proxy-role
+      - rules:
+        - nonResourceURLs:
+          - /metrics
+          verbs:
+          - get
+        serviceAccountName: {{ .OperatorName }}-metrics-reader
       deployments:
-      - name: {{ .OperatorName }}
+      - name: {{ .OperatorName }}-controller-manager
         spec:
           replicas: 1
           selector:
             matchLabels:
-              name: {{ .OperatorName }}
+              control-plane: controller-manager
           strategy: {}
           template:
             metadata:
               labels:
-                name: {{ .OperatorName }}
+                control-plane: controller-manager
             spec:
               containers:
-              - command:
-                - {{ .OperatorName }}
-                env:
-                - name: WATCH_NAMESPACE
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.annotations['olm.targetNamespaces']
-                - name: POD_NAME
-                  valueFrom:
-                    fieldRef:
-                      fieldPath: metadata.name
-                - name: OPERATOR_NAME
-                  value: {{ .OperatorName }}
-                image: {{ .TestImageTag }}
-                imagePullPolicy: Never
-                name: {{ .OperatorName }}
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
                 resources: {}
-              serviceAccountName: {{ .OperatorName }}
+              - args:
+                - --metrics-addr=127.0.0.1:8080
+                - --enable-leader-election
+                command:
+                - /manager
+                image: {{ .TestImageTag }}
+                name: manager
+                resources:
+                  limits:
+                    cpu: 100m
+                    memory: 30Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
+              terminationGracePeriodSeconds: 10
       permissions:
       - rules:
         - apiGroups:
           - ""
           resources:
-          - pods
-          - services
-          - endpoints
-          - persistentvolumeclaims
-          - events
           - configmaps
-          - secrets
           verbs:
-          - '*'
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
         - apiGroups:
           - ""
           resources:
-          - namespaces
+          - configmaps/status
```

Comment:
configmaps/status does not exist. See; https://github.com/kubernetes-sigs/kubebuilder/pull/1511

Is it updated manually or via the scaffold? If it is updated automatically and check it is fine because it will be fixed when we start to use the v3

Reply:
That is correct. I'm going to leave this as-is for now, we can change this when v3 is stabilized. Added a comment.